resources/participants.md

```diff
@@ -42,22 +42,33 @@ The response from the `participants` endpoint is a single fundraiser object.
 
 |Field|Type|Description|Guaranteed|Filterable|Notes|
 |---|---|---|---|---|---|
+|`activityGoal`|`float`|The participant's activity goal|No|No||
 |`avatarImageURL`|`string`|The URL for the avatar image associated with this Participant|Yes|Yes||
 |`campaignDate`|`date`|The date of the Personal Campaign|||Personal Campaigns only; ISO-8601 format|
 |`campaignName`|`string`|The name of the Personal Campaign|||Personal Campaigns only|
+|`canBeCaptain`|`boolean`|`true` if the participant's type permits them to be a team captain|No|No||
 |`createdDateUTC`|`date`|The date (in UTC) this Participant was created|Yes|Yes|ISO-8601 format|
 |`displayName`|`string`|This Participant's name|Yes|Yes||
 |`eventID`|`integer`|The ID of the Event this Participant is associated with|Yes|Yes||
 |`eventName`|`string`|The name of the Event this Participant is associated with|Yes|||
 |`fundraisingGoal`|`float`|This Participant's individual goal|Yes|Yes||
+|`hasActivityTracking`|`boolean`|`true` if participant's type has activity tracking enabled|No|No||
+|`isCheckedIn`|`boolean`|`true` if the participant has completed event check-in|No|Yes||
+|`isCustomAvatarImage`|`boolean`|`true` if that participant has uploaded an avatar image|No|No||
 |`isTeamCaptain`|`boolean`|`true` if this Participant is the captain of a team||`true`|Team Participants only|
 |`links`|`object`|An object containing related resources|||`donate`: The donation URL for this Participant<br />`page`: The URL for this Participant's Fundraising Page<br />`stream`: The URL for the Live Fundraising(TM) stream associated with this Participant|
+|`mustBeTeamCaptain`|`boolean`|`true` if the participant's type requires them to be a team captain|No|No||
 |`numAwardedBadges`|`integer`|The number of badges this Participant has received||Yes|Added: 1.2|
 |`numIncentives`|`integer`|The number of fundraising incentives this Participant has created||Yes|Added: 1.3|
 |`numMilestones`|`integer`|The number of fundraising milestones this Participant has created||Yes|Added: 1.3|
 |`numDonations`|`integer`|The number of donations this Participant has received|Yes|Yes||
 |`participantID`|`integer`|The unique ID of this Participant|Yes|Yes||
+|`participantTypeCode`|`string`|The code associated with the participant's type|Yes|Yes||
+|`streamingChannel`|`string`|The participant's streaming channel name|No|No||
```

Comment:
please note that we identify entities as proper nouns (ex: `Participant`)

Reply:
Gotcha. Think I got all the places where they're used that way.